### PR TITLE
:sparkles: add support for financial document v1.10 & invoice v4.8

### PR DESCRIPTION
**Pull request by sebastianMindee**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires a change to the official Guide documentation.

### CODE REVIEW

**Review by ianardee — CHANGES_REQUESTED** [review]
missing `is_computed` on date

File: spec/fields/date_field_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'mindee'
+
+describe Mindee::Parsing::Standard::StringField do
```
Comment (ianardee) [inline spec/fields/date_field_spec.rb:5]:
```suggestion
describe Mindee::Parsing::Standard::DateField do
```

File: spec/fields/date_field_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  describe 'constructor with no date' do
+    let(:field_dict) do
+      {
+        'confidence' => 0,
+        'polygon' => [],
+      }
+    end
+
+    it 'sets no date' do
+      field = Mindee::Parsing::Standard::StringField.new(field_dict)
```
Comment (ianardee) [inline spec/fields/date_field_spec.rb:38]:
```suggestion
      field = Mindee::Parsing::Standard::DateField.new(field_dict)
```

**Review by ianardee — CHANGES_REQUESTED** [review]

File: spec/fields/date_field_spec.rb (hunk 1 above)
Comment (sebastianMindee) [reply spec/fields/date_field_spec.rb:5]: :shushing_face:

**Review by ianardee — APPROVED** [review]

**Pull request merged**